### ISSUE TITLE
the result in LFPW and Helen

### ISSUE THREAD

**Issue by yyytq** [jwyang/face-alignment#26]
first of all  thank you for your code ! it works well !
but i notice that when i run it on LFPW, the err is much  higher than the paper(5.17/3.35). but on Helen, the result is nearly the same . how do you think about this issue?
the second one i'm confusion is that for helen (194 landmakers), the paper use total 300 trees, averaged less than 2 for each maker. how it comes???  it seems like only decision tree instead random forest~~~~(>_<)~~~~ 

looking forward to your reply~~~
thanks again!

**Comment by jwyang** [comment]
@yyytq ,

but i notice that when i run it on LFPW, the err is much higher than the paper(5.17/3.35). but on Helen, the result is nearly the same . how do you think about this issue?

>>Hi, on LFPW dataset, do you test also using 29 landmarks as used in the paper? If not, the error should be 5.00+

the second one i'm confusion is that for helen (194 landmakers), the paper use total 300 trees, averaged less than 2 for each maker. how it comes??? it seems like only decision tree instead random forest~~

>>Yes, I am also a little confused on that. it is indeed a little odd just using 300 trees for 194 landmarks, i think there are some tricks on that, it would be better to ask the authors, :)

**Comment by yyytq** [comment]
thank you very much!! 
I have tested 29 landmarkrs .  what i'm thinking about is that on LFPW the result is 5.00+/3.35, on helen is 5.5+/5.81,5.4    why the code perform much better on helen?  is only the reason of the tree number?

what's more, do you think the different raio_radius may influence  a lot ?

**Comment by jwyang** [comment]
Oh, I see, so what is the training sample you used for testing each
dataset. the tree number for these two sets are same to each other, right?
In this case, I think the size of training sample made the difference,
possibly

Not that much, I set them empirically actually

**Comment by yyytq** [comment]
i just use the trained model you have provided ! 
i means  the paper use less trees on helen, but we use the same trees number as LFPW . so if our result on helen is really  compareable with the paper ?

**Comment by jwyang** [comment]
yes, you are right, that is one of the difference. Another difference is that the model has trees with depth=4, while the paper has trees with depth=5, maybe you can modify the parameters to make them similar to the paper and have a try again.

**Comment by yyytq** [comment]
yeah, i have tried that. in fact, what  i want to do now is change the initial shape from meanshape to a shape that have been per-processed, so that the err can be reduced.  but i always can't get good result (>_<) 
do you think it is meaningful?

**Comment by jwyang** [comment]
it is a difficult thing, because at the begining, we do not know exactly
the pose and expression of a face. Actually, the first stage can be
regarded as a pre-processing step for the following face alignment. I also
did some experiment to estimate a more plausible initial shape before but
failed to imporve the performance. As a result, I do not think changing
another initial shape does not make much different to the final alignment
result.
